engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tags.controller.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2015 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public
+ * License as published by the Free Software Foundation; either version
+ * 2 of the License (GPLv2) or (at your option) any later version.
+ * There is NO WARRANTY for this software, express or implied,
+ * including the implied warranties of MERCHANTABILITY,
+ * NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+ * have received a copy of GPLv2 along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+*/
+
+/**
+ * @ngdoc object
+ * @name  Bastion.docker-tags.controller:DockerTagsController
+ *
+ * @requires $scope
+ * @requires $location
+ * @requires Nutupane
+ * @requires DockerTag
+ * @requires CurrentOrganization
+ *
+ * @description
+ *   Provides the functionality specific to docker tags for use with the Nutupane UI pattern.
+ *   Defines the columns to display and the transform function for how to generate each row
+ *   within the table.
+ */
+angular.module('Bastion.docker-tags').controller('DockerTagsController',
+    ['$scope', '$location', 'Nutupane', 'DockerTag', 'CurrentOrganization',
+    function ($scope, $location, Nutupane, DockerTag, CurrentOrganization) {
+
+        var params = {
+            'organization_id':  CurrentOrganization,
+            'search':           $location.search().search || "",
+            'sort_by':          'name',
+            'sort_order':       'ASC',
+            'paged':            true
+        };
+
+        var nutupane = new Nutupane(DockerTag, params);
+        $scope.table = nutupane.table;
+        $scope.removeRow = nutupane.removeRow;
+
+        $scope.table.closeItem = function () {
+            $scope.transitionTo('docker-tags.index');
+        };
```

Comment:
If you aren't planning to have a master-detail, you can remove this in favor of setting `nutupane.masterOnly = true`